test/ConnectTest.php

```diff
@@ -64,7 +64,19 @@ public function testUnknownHostConnect()
             $ldap->connect()->bind('CN=ignored,DC=example,DC=com', 'ignored');
             $this->fail('Expected exception for unknown host');
         } catch (Exception\LdapException $zle) {
-            $this->assertContains('Can\'t contact LDAP server', $zle->getMessage());
+            $alternatives = ['Can\'t contact LDAP server',
+                          'Failed to connect to LDAP server'];
+            $message = $zle->getMessage();
+            $found = false;
+
+            foreach($alternatives as $alternative){
+                if(strstr($message, $alternative)){
+                    $found = true;
+                    break;
+                }
+            }
+
+            $this->assertTrue($found, 'Found one of the expected failure messages');
```

Comment:
replace this with a ```$this->fail('Didn't find an expected failure message')```.

When a message is found we return early and this part will never be reached, but when no message is found this will be reached and the test fails.